minvws/nl-kat-coordination
# Do not let enabling plugins affect the global plugin cache

### Changes

A nice example of why you should avoid caches as long as possible.

This PR makes sure we do not edit the `enabled` field in-place, since these objects are cached in-memory and hence take "remember" their previous state. If we perform no overrides by default, the state transcends organisations.

### Issue link

Closes https://github.com/minvws/nl-kat-coordination/issues/3941

### Demo

See the new integration test that fails on main.

### QA notes

Please check if enabling `dns-records` in organization 1 does not enable it for organization 2!

---

### Code Checklist




---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
